### PR TITLE
Feature/bitrise

### LINKED ISSUE #2891: Final Bitrise Migration
Enable the final features in Bitrise for RC candidates after v2.6.0

- Release Build and Store Staging
- E2E Test Execution on a Nightly Basis
- Test Reporting & Console Access (optional)

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**

Remove CircleCI and add Bitrise for release

**Checklist**

* [X] There is a related GitHub issue
* [X] Tests are included if applicable
* [X] Any added code is fully documented

**Issue**

Resolves #2891

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildscript {
 
 	ext {
 		buildToolsVersion = "29.0.2"
-		minSdkVersion = 19
+		minSdkVersion = 26
```
Comment (sethkfman) [inline android/build.gradle:7]: Need to confirm this is acceptable

File: scripts/bump-version.sh [hunk 2]
```diff
@@ -5,32 +5,39 @@ set -u
...
-        version=$major'.'$minor'.'$patch
-    elif [[ $1 == *"-minor"* ]] ; then
-        minor=$((MINOR +  1));
-        patch="0";
-        version=$MAJOR'.'$minor'.'$patch
-    elif [[ $1 == *"-patch"* ]] ; then
-        patch=$((PATCH +  1));
-        version=$MAJOR'.'$MINOR'.'$patch
-    fi
+# if [[ $1 == *"release/"* ]] ; then
```
Comment (rickycodes) [inline scripts/bump-version.sh:15]: do we want to leave comments? everything is in git, so imo we don't need to do this?

File: scripts/build.sh [hunk 3]
```diff
@@ -164,7 +168,7 @@ buildIosSimulator(){
 
 buildIosSimulatorE2E(){
 	prebuild_ios
-	xcodebuild -workspace ios/MetaMask.xcworkspace -scheme MetaMask -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build
+	cd ios && xcodebuild -workspace MetaMask.xcworkspace -scheme MetaMask -configuration Debug -sdk iphonesimulator -derivedDataPath build
```
Comment (rickycodes) [inline scripts/build.sh:171]: is this change necessary? why cd into the directory first?

**Review by rickycodes — COMMENTED** [review]
left a couple comments, but I am seeing a lot of changes here that don't seem necessary? (like the re-tabbing in `bitrise.yml`). we should probably have a task that automatically formats that file so we don't have such large changes...

File: scripts/build.sh (hunk 3 above)
Comment (sethkfman) [reply scripts/build.sh:171]: It is. Just a preference to shorten the `xcodebuild`

File: scripts/bump-version.sh (hunk 2 above)
Comment (sethkfman) [reply scripts/bump-version.sh]: Resolved

**Review by rickycodes — APPROVED** [review]
